scripts/prepare_network.py

```diff
@@ -225,18 +225,18 @@ def set_transmission_limit(n, ll_type, factor, costs, Nyears=1):
 
 
 def average_every_nhours(n, offset):
-    logger.info(f"Resampling the network to {offset}")
+    logger.info(f"Resampling the network to {offset.casefold()}")
```

Comment:
My feeling is that it'd be better to keep here the capital `H` to be consistent with the options definition in the file name. The point is, snapshots resampling is coded in the model name as something like `24H` (so called [opts wildcard](https://pypsa-eur.readthedocs.io/en/latest/wildcards.html#the-opts-wildcard)). So, that is something which is visible for a modeller, and as a such I'd expect that capitalisation would be kept also in the logs.

@finozzifa does it sound reasonable for you?

Reply:
I agree! Done and thanks :)